projects/dendrites/permutedMNIST/experiments/base.py

```diff
@@ -140,19 +135,19 @@ class PermutedMNISTExperiment(mixins.RezeroWeights,
         input_size=784,
         output_size=10,
         hidden_sizes=[2048, 2048],
-        num_segments=10,    # NUM_TASKS
+        num_segments=10,  # NUM_TASKS
         dim_context=1024,
         kw=True,
         kw_percent_on=tune.sample_from(
-            lambda spec: np.random.choice([0.05, 0.1, 0.2, 0.0])),
+            lambda spec: np.random.choice([0.05, 0.1, 0.2, 0.0])
+        ),
         dendrite_weight_sparsity=0.0,
         weight_sparsity=tune.sample_from(
-            lambda spec: np.random.choice([0.90, 0.8, 0.7, 0.5, 0.25, 0.0])),
+            lambda spec: np.random.choice([0.90, 0.8, 0.7, 0.5, 0.25, 0.0])
+        ),
     ),
-
     epochs=tune.sample_from(lambda spec: np.random.randint(1, 3)),
     num_samples=5,
-
```

Comment:
Why undo the blank line removal? They help for readability.

Reply:
automatic formatting using black and isort.